### PR TITLE
[zh-cn]: create the translation of `documentpictureinpictureevent`

### PR DESCRIPTION
**Pull request by tianyeeT**
### Related issues and pull requests
part of #23557

### CODE REVIEW

**Comment by skyclouds2001** [comment]
Note this may be blocked by https://github.com/mdn/content/pull/35918

File: files/zh-cn/web/api/documentpictureinpictureevent/index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+{{Specifications}}
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 参见
+
+- {{domxref("Document Picture-in-Picture API", "Document Picture-in-Picture API", "", "nocode")}}
+- [使用文档画中画 API](/zh-CN/docs/Web/API/Document_Picture-in-Picture_API/Using)
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/documentpictureinpictureevent/index.md:65]: 考虑到整体 API 名称都不翻译，这里也不翻译更合适？

Comment (tianyeeT) [inline files/zh-cn/web/api/documentpictureinpictureevent/index.md:65]:
```suggestion
- [使用 DocumentPictureInPictureEvent API](/zh-CN/docs/Web/API/Document_Picture-in-Picture_API/Using)
```

File: files/zh-cn/web/api/documentpictureinpictureevent/index.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+---
+title: DocumentPictureInPictureEvent
+slug: Web/API/DocumentPictureInPictureEvent
+l10n:
+  sourceCommit: 89c435da452257b944b403cc9e45036fcb22590e
+---
+
+{{APIRef("Document Picture-in-Picture API")}}{{SeeCompatTable}}{{SecureContext_Header}}
+
+**`DocumentPictureInPictureEvent`** 接口是 {{domxref("Document Picture-in-Picture API", "Document Picture-in-Picture API", "", "nocode")}} 的事件对象，表示当画中画（Picture-in-Picture）窗口打开时触发的 {{domxref("DocumentPictureInPicture/enter_event", "enter")}} 事件。
```
Comment (yin1999) [inline files/zh-cn/web/api/documentpictureinpictureevent/index.md:10]:
```suggestion
**`DocumentPictureInPictureEvent`** 接口是 {{domxref("Document Picture-in-Picture API", "Document Picture-in-Picture API", "", "nocode")}} 的 {{domxref("DocumentPictureInPicture/enter_event", "enter")}} 事件（会在画中画窗口打开时触发）的对象。
```

File: files/zh-cn/web/api/documentpictureinpictureevent/index.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+{{InheritanceDiagram}}
+
+## 构造函数
+
+- {{domxref("DocumentPictureInPictureEvent.DocumentPictureInPictureEvent", "DocumentPictureInPictureEvent()")}} {{Experimental_Inline}}
+  - : 创建一个新的 `DocumentPictureInPictureEvent` 对象实例。
+
+## 实例方法
+
+_继承自其父类 {{DOMxRef("Event")}} 的方法。_
```
Comment (yin1999) [inline files/zh-cn/web/api/documentpictureinpictureevent/index.md:21]: “方法继承自父接口”/“继承其父接口的方法”，这里就不应该加“自”字

```suggestion
_继承其父接口 {{DOMxRef("Event")}} 的方法。_
```

File: files/zh-cn/web/api/documentpictureinpictureevent/index.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+## 实例属性
+
+_继承自其父类 {{DOMxRef("Event")}} 的属性。_
```
Comment (yin1999) [inline files/zh-cn/web/api/documentpictureinpictureevent/index.md:25]:
```suggestion
_继承其父接口 {{DOMxRef("Event")}} 的属性。_
```

File: files/zh-cn/web/api/documentpictureinpictureevent/index.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+- {{domxref("DocumentPictureInPictureEvent.window", "window")}} {{ReadOnlyInline}} {{Experimental_Inline}}
+  - : 返回一个 {{domxref("Window")}} 实例，表示事件触发时的画中画窗口的浏览上下文。
```
Comment (yin1999) [inline files/zh-cn/web/api/documentpictureinpictureevent/index.md:28]:
```suggestion
  - : 返回一个 {{domxref("Window")}} 实例，表示触发事件的 `DocumentPictureInPicture` 窗口的浏览上下文。
```

File: files/zh-cn/web/api/documentpictureinpictureevent/index.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+{{Specifications}}
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 参见
+
+- {{domxref("Document Picture-in-Picture API", "Document Picture-in-Picture API", "", "nocode")}}
+- [使用 DocumentPictureInPictureEvent API](/zh-CN/docs/Web/API/Document_Picture-in-Picture_API/Using)
```
Comment (yin1999) [inline files/zh-cn/web/api/documentpictureinpictureevent/index.md:65]: 麻烦对一下

```suggestion
- [使用 Document Picture-in-Picture API](/zh-CN/docs/Web/API/Document_Picture-in-Picture_API/Using)
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

**Review by yin1999 — APPROVED** [review]

**Review by yin1999 — APPROVED** [review]

**Pull request merged**